packages/cli/__tests__/zosjobs/__unit__/submit/Submit.shared.handler.unit.test.ts

```diff
@@ -20,24 +20,40 @@ import {
 import { mockHandlerParameters } from "@zowe/cli-test-utils";
 
 process.env.FORCE_COLOR = "0";
+let DEFAULT_PARAMETERS: IHandlerParameters;
+let USSFILE_PARAMETERS: IHandlerParameters;
+let LOCALFILE_PARAMETERS: IHandlerParameters;
 
-const DEFAULT_PARAMETERS: IHandlerParameters = mockHandlerParameters({
-    arguments: UNIT_TEST_ZOSMF_PROF_OPTS,
-    positionals: ["zos-jobs", "submit", "data-set"],
-    definition: SubmitDefinition.SubmitDefinition,
-    profiles: UNIT_TEST_PROFILES_ZOSMF
-});
+describe("submit shared handler", () => {
+    beforeEach(() => {
+        DEFAULT_PARAMETERS = mockHandlerParameters({
+            arguments: UNIT_TEST_ZOSMF_PROF_OPTS,
+            positionals: ["zos-jobs", "submit", "data-set"],
+            definition: SubmitDefinition.SubmitDefinition,
+            profiles: UNIT_TEST_PROFILES_ZOSMF
+        });
 
-const LOCALFILE_PARAMETERS: IHandlerParameters = mockHandlerParameters({
-    arguments: UNIT_TEST_ZOSMF_PROF_OPTS,
-    positionals: ["zos-jobs", "submit", "local-file"],
-    definition: SubmitDefinition.SubmitDefinition,
-    profiles: UNIT_TEST_PROFILES_ZOSMF
-});
+        USSFILE_PARAMETERS = mockHandlerParameters({
+            arguments: UNIT_TEST_ZOSMF_PROF_OPTS,
+            positionals: ["zos-jobs", "submit", "uss-file"],
+            definition: SubmitDefinition.SubmitDefinition,
+            profiles: UNIT_TEST_PROFILES_ZOSMF
+        });
 
-describe("submit shared handler", () => {
+        LOCALFILE_PARAMETERS = mockHandlerParameters({
+            arguments: UNIT_TEST_ZOSMF_PROF_OPTS,
+            positionals: ["zos-jobs", "submit", "local-file"],
+            definition: SubmitDefinition.SubmitDefinition,
+            profiles: UNIT_TEST_PROFILES_ZOSMF
+        });
+    });
+
+    afterEach(() => {
+        jest.resetAllMocks();
+    });
 
     describe("error handling", () => {
```

Comment:
Thanks for boosting coverage for `submit data-set` and `submit local-file` too 🙂